contracts/token/ERC721/IERC721Receiver.sol

```diff
@@ -6,7 +6,7 @@ pragma solidity ^0.4.24;
  * @dev Interface for any contract that wants to support safeTransfers
  * from ERC721 asset contracts.
  */
-contract ERC721Receiver {
+contract IERC721Receiver {
   /**
    * @dev Magic value to be returned upon successful reception of an NFT
    *  Equals to `bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))`,
```

Comment:
line below this should be `IERC721Receiver(address(0)).onERC721Received.selector`